### ISSUE TITLE
Insider reflow validation: Overview summary bar not readable by screen reader

### ISSUE THREAD

**Issue by pownkel** [microsoft/accessibility-insights-web#3083]
## Describe the bug

When reading the Overview page with NVDA, the summary bar is read as three percentages with no indication of what the numbers represent (passed, incomplete, and failed requirements). The outcome summary bar should have a meaningful accessible name to communicate this information.

**To Reproduce**

1. Start a new Assessment
2. Using NVDA, read or nagivate through the page until you reach the "summary" heading
3. Press the down arrow to read the next line. The summary bar will be read as something like "7%93%0%"

## Expected behavior

NVDA reads something more meaningful for the summary bar

## Context (please complete the following information)

- AI-Web Version & Environment: Insider 2.19.0
